### PR TITLE
update connection pooling doc

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

File: src/docs/user/references/DataConnectors.adoc [hunk 1]
```diff
@@ -234,64 +234,85 @@ In the import mode, the output containing the imported data takes one or many of
...
 we have implemented the connection pooling feature to any external SQL database based on the https://github.com/brettwooldridge/HikariCP[HikariCP^] Framework.
-This feature is showcased through two workflow templates: SQL_Pooled_Connection_Query and SQL_Pooled_Connection_Update.
-The SQL_Pooled_Connection_Query workflow is based on the replicate control flow where the split task is Open_Pooled_Connection, the process task is Execute_Query_Statements and finally the merge task is Merge_Result_Set.
-The SQL_Pooled_Connection_Update workflow is composed of two tasks: Open_Pooled_Connection and Execute_Update_Statements.
+This feature is showcased through two workflow templates: SQL_Pooled_Connection_Query and SQL_Pooled_Connection_Update. Both are a single task workflow.
 
 *Variables:*
 
-Both workflows share the same list of variables. Consequently, we describe them in the same following table.
+The SQL_Pooled_Connection_Update workflow shares with the SQL_Pooled_Connection_Query workflow the same list of variables. Consequently, we describe them in the same following table.
```
Comment (luispinedamx) [inline src/docs/user/references/DataConnectors.adoc]: I find this sentence confusing. "... the same following table" is not grammatically correct. A suggestion:

The variables listed in the following table [exist for both | are common to] the SQL_Pooled_Connection_Update and the SQL_Pooled_Connection_Query workflows.

Comment (aminelouati) [reply src/docs/user/references/DataConnectors.adoc]: Thanks bro ;) I take it blindly :D

File: src/docs/user/references/DataConnectors.adoc [hunk 2]
```diff
@@ -234,64 +234,85 @@ In the import mode, the output containing the imported data takes one or many of
 * in a _JSON_ format using the variable `DATAFRAME_JSON`.
 
 ==== Database Connections Pooling
-To allow the persistence of a single connection across multiple SQL tasks (basically when running several Tasks concurrently on the same machine),
+To allow the persistence of a single connection across multiple SQL tasks (basically when running several tasks concurrently on the same machine),
 we have implemented the connection pooling feature to any external SQL database based on the https://github.com/brettwooldridge/HikariCP[HikariCP^] Framework.
-This feature is showcased through two workflow templates: SQL_Pooled_Connection_Query and SQL_Pooled_Connection_Update.
-The SQL_Pooled_Connection_Query workflow is based on the replicate control flow where the split task is Open_Pooled_Connection, the process task is Execute_Query_Statements and finally the merge task is Merge_Result_Set.
-The SQL_Pooled_Connection_Update workflow is composed of two tasks: Open_Pooled_Connection and Execute_Update_Statements.
+This feature is showcased through two workflow templates: SQL_Pooled_Connection_Query and SQL_Pooled_Connection_Update. Both are a single task workflow.
```
Comment (mboussaa) [inline src/docs/user/references/DataConnectors.adoc]: -Both are single-task workflows
-don't like "showcased". => demonstrated maybe

File: src/docs/user/references/DataConnectors.adoc [hunk 3]
```diff
@@ -234,64 +234,85 @@ In the import mode, the output containing the imported data takes one or many of
...
 
 *Variables:*
 
-Both workflows share the same list of variables. Consequently, we describe them in the same following table.
+The variables listed in the following table exist for both the SQL_Pooled_Connection_Update and the SQL_Pooled_Connection_Query workflows.
```
Comment (mboussaa) [inline src/docs/user/references/DataConnectors.adoc]: for both, the SQL_Pooled_Connection_Update and the SQL_Pooled_Connection_Query workflows.
OR
for both SQL_Pooled_Connection_Update and SQL_Pooled_Connection_Query workflows.

Up to you.

File: src/docs/user/references/DataConnectors.adoc [hunk 4]
```diff
@@ -234,64 +234,85 @@ In the import mode, the output containing the imported data takes one or many of
...
+| `OUTPUT_TYPE`
+| If set to `HTML`, it allows to preview the results in the Scheduler Portal in a HTML format. If set to `CSV`, it allows to download the results from the Scheduler Portal in a CSV format.
+| Workflow, Task
+| Yes
+| String
+| Values: `{CSV, HTML}`.
+
+Default: `HTML`
+| `STORE_RESULT_VARIABLE`
+| if not empty, the value will be the name of a variable that contains the resultSet (converted into a List of Maps) of the query. This variable can be used in other tasks.
```
Comment (mboussaa) [inline src/docs/user/references/DataConnectors.adoc]: the resultSet of the query (converted into a List of Maps)? maybe better for readability

**Pull request merged**